torch/csrc/jit/runtime/register_prim_ops_c10.cpp

```diff
@@ -33,4 +34,17 @@ static auto registry_prim =
             torch::RegisterOperators::options()
                 .catchAllKernel(
                     [](Scalar scalar) -> int64_t { return scalar.toInt(); })
+                .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA))
+        .op("aten::Int.str(str a) -> int",
+            torch::RegisterOperators::options()
+                .catchAllKernel(
+                    [](std::string str) -> int64_t {
+                        std::string::size_type sz;
+                        int64_t val = static_cast<int64_t>(c10::stoll(str, &sz));
+                        if (sz != str.size()) {
+                            throw std::runtime_error(
+                                "int() only accepts a string of a single integer");
```

Comment:
Can you have this match Python more closely? Something like

`invalid literal for int() with base 10: ' + str + "'"`

Reply:
Modified the error messages for both `int()` and `float()`.